app/public/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "short_name": "Winds",
+  "name": "Winds - Powered by GetStream.io",
+  "description": "Winds 3.0 is a beautiful Open Source Podcast & RSS Reader created by Stream. Free desktop apps are available for macOS, Linux and Windows. A web version is available as well.",
+  "icons": [
+      {
+          "src": "/favicons/favicon-96x96.png",
+          "type": "image/png",
+          "sizes": "96x96"
+      },
+      {
+          "src": "/favicons/favicon-128x128.png",
+          "type": "image/png",
+          "sizes": "128x128"
+      },
+      {
+          "src": "/favicons/favicon-196x196.png",
+          "type": "image/png",
+          "sizes": "196x196"
+      },
+      {
+          "src": "/favicons/favicon-256x256.png",
+          "type": "image/png",
+          "sizes": "256x256"
+      },
+      {
+          "src": "/favicons/favicon-512x512.png",
+          "type": "image/png",
+          "sizes": "512x512"
+      }
+  ],
+  "start_url": "/?source=pwa",
```

Comment:
Note: the right way to track the PWA usage into Google Analytics is by adding `utm_source`, not `source`